### PR TITLE
[BD-46] refactor: refactoring buttons variants

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
### **Description**
- refactored Button variants
- refactored overrides.scss file

Issue: https://github.com/openedx/paragon/issues/2389

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.


Thanks for your pull request, but this repo does not accept outside contributions unless they are under contract. If you think this is an error, please contact @nedbat.

**Review by monteri — APPROVED** [review]

**Comment by adamstankiewicz** [comment]
[inform] Note: the failing `openedx/cla` check is OK. This work is under contract with Raccoon Gang (BD-46).

File: tokens/src/themes/light/components/Image.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "elevation": {
     "image": {
       "thumbnail": {
-        "box-shadow": { "value": "none" }
+        "box-shadow": { "value": "0 1px 2px rgba(0, 0, 0, .075)" }
```
Comment (adamstankiewicz) [inline tokens/src/themes/light/components/Image.json:5]: [clarification] Looking at the `Image` component using the edX.org theme on the Paragon docs site for `master`, it looks like the `Image` thumbnail doesn't have a `box-shadow`?

image

Comment (PKulkoRaccoonGang) [reply tokens/src/themes/light/components/Image.json:5]: Yes you are right
This pull request will be closed, and a new one with corrections will be opened [here](https://github.com/edx/brand-edx.org/pull/82)

**Pull request closed without merging**